packages/charts/chart-utilities/src/PlotlySchemaConverter.ts

```diff
@@ -646,7 +646,7 @@ export const mapFluentChart = (input: any): OutputChartType => {
 };
 
 const canMapToGantt = (data: Partial<PlotData>) => {
-  return isDateArray(data.base) || isNumberArray(data.base);
+  return isDateArray(data.base) || isNumberArray(data.base) || isDate(data.base) || isNumber(data.base);
```

Comment:
Since we are updating in chart-utilities we would need to fix downstream in v8 also